docs/en/docs/middleware/mysql/faq/faq-others.md

```diff
@@ -12,18 +18,19 @@ The database is running normally, but an error occurs when creating a database u
     [root@master-01 ~]$ kubectl get secret -n mcamel-system mcamel-common-mysql-cluster-secret -o=jsonpath='{.data.ROOT_PASSWORD}' | base64 -d
     ```
 
-2. If the password contains special characters such as __-__ , attempting to enter the original password in MySQL Shell will result in the following error:
+2. If the password contains special characters such as __-__ , attempting to enter the original password 
+   in MySQL Shell will result in the following error:
 
     ```console
     bash-4.4# mysql -uroot -p
     Enter password:
     ERROR 1045 (28000): Access denied for user 'root'@'localhost' (using password: YES)
     ```
 
-3. To resolve this issue, perform the following steps:
+3. To resolve this issue, you are offered two methods to clean up and rebuild:
 
-    - Method 1: Clean up and rebuild:
-        - Clear the data directory by deleting the Pod. Wait for the sidecar to be running, then delete the data directory again and finally delete the Pod:
+    - Method 1: Clear the data directory and delete the Pod. Wait for the sidecar to be running, 
+      then delete the data directory again and repete deleting the Pod:
 
             ```bash
```

Comment:
The [preview](https://deploy-preview-5274--daocloud-docs.netlify.app/en/middleware/mysql/faq/faq-others#error-encountered-when-creating-database-with-cr) looks not good.